vcr/pe/test/pd_jsonld_jwt.json

```diff
@@ -0,0 +1,99 @@
+{
+  "id": "Definition requesting NutsOrganizationCredential",
```

Comment:
All examples online define multiple paths to handle the differente between jsonld and jwt. Would be best to add a test for that as well.

Reply:
Couldn't find them, and pick sounded weird to me (but works), will do